src/Container/ShopifyClientFactory.php

```
@@ -51,18 +50,6 @@ public function __invoke(ContainerInterface $container)
             throw new RuntimeException('Container config does not have a "zfr_shopify" key');
         }
 
-        $config = $config['zfr_shopify'];
-
-        if ($config['private_app']) {
-            if (!isset($config['shop']) || !isset($config['api_key']) || !isset($config['password'])) {
-                throw new RuntimeException('Options "shop", "api_key" and "password" are mandatory when creating Shopify client for a private app');
-            }
-        } else {
-            if (!isset($config['api_key']) || !isset($config['shared_secret'])) {
-                throw new RuntimeException('Options "api_key" and "shared_secret" are mandatory when creating Shopify client for a public app');
-            }
-        }
-
-        return new ShopifyClient($config);
+        return new ShopifyClient($config['zfr_shopify']);
```

Should throw exception if `zfr_shopify` is undefined index